roles/candlepin/defaults/main.yml

```diff
@@ -12,11 +12,13 @@ candlepin_ciphers:
   - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
   - TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256
   - TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256
-candlepin_container_image: quay.io/ehelms/candlepin
-candlepin_container_tag: 4.4.14
+candlepin_container_image: quay.io/candlepin/candlepin
+candlepin_container_tag: 4.4.20-1
 
 candlepin_db_host: localhost
 candlepin_db_port: 5432
 candlepin_db_ssl: false
 candlepin_db_ssl_ca: None
 candlepin_db_ssl_verify: true
+
+candlepin_tomcat_home: /opt/tomcat/conf
```

Comment:
what was the reason for this change? Once I undid this change and let the config files be under `/etc/tomcat` then it worked

Reply:
I think I ran into errors originally with this path. And then I assumed since tomcat is in /opt/tomcat and there is a `conf` directory in there that it should use that. Since it wouldn't want to clash with system tomcat from package.

What is the expectation for where configuration files should live with this image?

Reply:
I tested this locally and got a green result. Pushed update to this PR and we'll see what automation says.

Reply:
I see in the candlepin image we build we use /opt/tomcat/conf, so I have no idea why that doesn't work. I'll try to tinker a bit more on monday to see why

Reply:
@ehelms so it looks like the reason it worked previously by using /etc/tomcat/conf was because the wrong container was still being used (the one with the rpms). I had to do a few changes to get it working. Should I push a commit to this PR?

Reply:
You can find the diff here: https://github.com/nikosmoum/foreman-quadlet/commit/94d127d73401d30aef6eb3d6c656c6682834ab71